### ISSUE TITLE
Could the contigs.fasta and unassembled.fasta be merged?

### ISSUE THREAD

**Issue by SarahIsme** [marbl/canu#2102]
Hello, as discussed in #2100, single reads can end up in the unassembled sequences so they have much more redundancy than the contigs. 

Although setting `contigFilter="2 0 1.0 0.5 0"` to disable contigFilter, the unassembled bases are more than the contigs.

So could I merge the contigs.fasta and unassembled.fasta to perform binning? Because single reads can end up in the unassembled sequences.

Thanks in advance!

**Comment by skoren** [comment]
I don't think you want single unassembled reads in your results. They aren't going to look like contigs (lower quality, likely have errors, no coverage info). I doubt binning can do anything reasonable with single reads. Any real genome composed of these single reads is very low coverage so likely can't be assembled anyway. 

At least, I'd first map the unassembled reads to the contigs first and eliminate any with reasonable mappings. They may map but had more error than the assembler tolerated. It should get rid of most of the redundancy and, if there really are unique sequences there, you could then include them. Though, as I said above, I generally don't include anything composed of a single read for downstream analysis.

**Comment by SarahIsme** [comment]
Thanks for your answer!
